Wiki-style-guide.md

```diff
@@ -86,3 +112,13 @@
   > * **Exploration:** An interactive lesson on Oppia.
 
 * Include punctuation at the ends of sentences. Also include punctuation at the ends of phrases that are exceptions to the "Write in complete sentences" rule above. For example, notice the period at the end of the example above.
+
+* Start each page with a bulletted table of contents that follows these rules:
```

Comment:
I think it's "bulleted", not "bulletted".

Reply:
fixed